### PR TITLE
Add CryptoAccountLinks schema

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: src/schemas/CryptoAccountLinks.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "patternProperties": {
+    "^[a-zA-Z0-9]{1,63}@[-a-zA-Z0-9]{3,16}:[-a-zA-Z0-9]{1,47}": {
+      "type": "string",
+      "pattern": "^ceramic://.+",
+      "maxLength": 1024
+    }
+  },
+  "propertyNames": {
+    "maxLength": 1024
+  },
```
Comment (oed) [inline src/schemas/CryptoAccountLinks.json:14]: What does this do?

**Review by oed — APPROVED** [review]
lgtm

File: src/schemas/CryptoAccountLinks.json (hunk 1 above)
Comment (ukstv) [reply src/schemas/CryptoAccountLinks.json:14]: Makes the schema "secure" according to ajv. Mandates that properties have limited length.

Comment (oed) [reply src/schemas/CryptoAccountLinks.json:14]: But maxLength was also set in the patternProperty above. Why does it need to be in both places?

Comment (ukstv) [reply src/schemas/CryptoAccountLinks.json:14]: One is for property name (`propertyNames` clause), another is for property value (`patternProperties/*`).

Comment (ukstv) [reply src/schemas/CryptoAccountLinks.json:14]: Both have to be explicitly constrained, as both are patterns.

Comment (oed) [reply src/schemas/CryptoAccountLinks.json:14]: Aha, I see. Thanks!

**Comment by PaulLeCam** [comment]
Thanks!

**Comment by ukstv** [comment]
@PaulLeCam @oed what's the release cycle here? Does the version here have to correspond to the version of Ceramic?

**Comment by PaulLeCam** [comment]
No the version here is not linked to Ceramic and the release cycle is independent from it, it's mostly about being in sync with `js-idx`.

**Pull request merged**